bedrock/firefox/templates/firefox/new/email/scene2.html

```diff
@@ -11,6 +17,11 @@
   <link rel="canonical" href="{{ settings.CANONICAL_URL }}/{{ LANG }}/firefox/new/">
 {% endblock %}
 
+{% block page_image %}{{ static('img/firefox/template/page-image.png') }}{% endblock %}
```

Comment:
Rather than add all these pieces of metadata on a per-page basis, would it make sense to create a `firefox/base-protocol.html` file that this template extends? (similar to how we already do for pebbles). That way all new Protocol pages under `/firefox/` could get this meta data for free?